### DISCUSSION TITLE
BepInEx while running under Proton

### DISCUSSION THREAD

**Discussion by gbd360** [BepInEx/BepInEx#345] [Help and Q&A]
I have tried to follow various instructions to install while running a game through Steam proton.  I cannot seem to get any version to work with the game.  Is there a way to start debugging the lack of loading Bepinex in this environment?  I don't have any information to share in which to open a bug report due to the failure nature.  I am running steam under Ubuntu 20.04.3.  Game launches fine, but Bepinex never seems to load.  It never even auto generates the directories.

**Comment by gbd360** [comment]
Finally found a guide.

Install protontricks:
https://github.com/Matoking/protontricks

Follow instructions here:
https://github.com/risk-of-thunder/R2Wiki/wiki/Getting-BepInEx-Console-Working-on-Linux

Install BepinEx normally

**Comment by Windows10CE** [comment]
Didn't see this before, but instructions for games running under proton/wine exist at https://docs.bepinex.dev/v5.4.16/articles/advanced/steam_interop.html#5-configure-bepinex-to-suit-your-needs, would probably be nice to have something in the installation docs point Linux users there.

**Comment by Kabouik** [comment]
I followed the instructions to [run BepInEx with Proton](https://docs.bepinex.dev/v5.4.16/articles/advanced/proton_wine.html) but somehow BepInEx still does not work: I modded KSP2 using CKAN, so all dependencies should be resolved, and I see no mods in game. 

![ss20240205-005342](https://github.com/BepInEx/BepInEx/assets/7107523/bb318379-ffdf-4712-852d-581b31d04095)

This is what the game folder looks like (note the `winhttp.dll`):

```
.
├── BepInEx
├── CKAN
├── doorstop_config.ini
├── Ksp2.log
├── KSP2_x64_Data
├── KSP2_x64.exe
├── MonoBleedingEdge
├── PDLauncher
├── preloader_20240204_224415_478.log
├── preloader_20240204_224654_694.log
├── preloader_20240204_231312_681.log
├── preloader_20240204_232255_682.log
├── preloader_20240205_003556_304.log
├── preloader_20240205_004322_850.log
├── preloader_20240205_004540_646.log
├── UnityCrashHandler64.exe
├── UnityPlayer.dll
└── winhttp.dll
```

And `BepInEx/`:

```
BepInEx/
├── core
├── patchers
└── plugins
```

Anything I can do to troubleshoot why BepInEx is not kicking in?

**Reply by Kabouik** [reply]
Alright this was a permission issue, due to CKAN being installed via Docker.
